P5/Source/Guidelines/en/CC-LanguageCorpora.xml

```diff
@@ -1077,53 +1080,48 @@ determined by an analysis of linguistic features of the text, excluding
+in these Guidelines, for example in chapters <ptr target="#CO"/> and
 <ptr target="#DS"/>.
 The contextual
 properties of a TEI text are fully documented in the TEI header, which
-is discussed in chapter <ptr target="#HD"/>, and in section <ptr target="#CCAH"/> of the present chapter.
- </p>
+is discussed in chapter <ptr target="#HD"/>, and in section <ptr target="#CCAH"/> of the present chapter.</p>
 <p>Other forms of linguistic annotation may be applied at a number of
-levels in a text.  A code (such as a word-class or part-of-speech
+levels in a text. A code (such as a word-class or part-of-speech
 code) may be associated with each word or token, or with groups of such
-tokens, which may be continuous, discontinuous, or nested.  A code may
+tokens, which may be continuous, discontinuous, or nested. A code may
 also be associated with relationships (such as cohesion) perceived as
-existing between distinct parts of a text.  The codes themselves may
+existing between distinct parts of a text. The codes themselves may
 stand for discrete non-decomposable categories, or they may represent
-highly articulated bundles of textual features.  Their function may be
+highly articulated bundles of textual features. Their function may be
 to place the annotated part of the text somewhere within a narrowly
 linguistic or discoursal domain of analysis, or within a more general
-semantic field, or any combination drawn from these and other domains.
- </p>
+semantic field, or any combination drawn from these and other domains.</p>
 <p>The manner by which such annotations are generated and attached to
-the text may be entirely automatic, entirely manual, or a mixture.  The
+the text may be entirely automatic, entirely manual, or a mixture. The
 ease and accuracy with which analysis may be automated may vary with the
-level at which the annotation is attached.  The method employed should
+level at which the annotation is attached. The method employed should
 be documented in the <gi>interpretation</gi> element within the encoding
-description of the TEI header, as described in section <ptr target="#HD53"/>.  Where different parts of a corpus have used different
+description of the TEI header, as described in section <ptr target="#HD53"/>. Where different parts of a corpus have used different
 annotation methods, the <att>decls</att> attribute should be used to
-indicate the fact, as further discussed in section <ptr target="#CCAS"/>.
- </p>
+indicate the fact, as further discussed in section <ptr target="#CCAS"/>.</p>
 <p>An extended example of one form of linguistic analysis commonly
-practised in corpus linguistics is given in section <ptr target="#AILA"/>.
-</p>
+practised in corpus linguistics is given in section <ptr target="#AILA"/>.</p>
 </div></div>
 <div type="div2" xml:id="CCREC"><head>Recommendations for the Encoding of Large Corpora</head>
 <p>These Guidelines include proposals for the identification and
```

Comment:
Suggestion for rewording line 1118 / 1117 "behind this catholic approach", perhaps rephrase using 'universal' or 'inclusive' instead?

Reply:
Seconded, and I'd suggest "universal"

Reply:
Changed to “universal” in 7f52ad0fa.